### PR TITLE
Add optional :meta field

### PR DESCRIPTION
**Pull request by mchaisse**

### CODE REVIEW

File: app/models/shortener/shortened_url.rb [hunk 1]
```diff
@@ -25,33 +25,46 @@ def self.clean_url(url)
...
-                            fresh:      fresh
-                          )
+                  owner:      owner,
+                  custom_key: custom_key,
+                  expires_at: expires_at,
+                  fresh:      fresh,
+                  meta:       meta)
       end
     else
       scope = owner ? owner.shortened_urls : self
```
Comment (ghost) [inline app/models/shortener/shortened_url.rb]: Move this to `apply_scope`

**Review by ghost — CHANGES_REQUESTED** [review]
minimal change

File: app/models/shortener/shortened_url.rb [hunk 2]
```diff
@@ -25,33 +25,46 @@ def self.clean_url(url)
   # generate a shortened link from a url
   # link to a user if one specified
   # throw an exception if anything goes wrong
-  def self.generate!(destination_url, owner: nil, custom_key: nil, expires_at: nil, fresh: false)
+  def self.generate!(destination_url, owner: nil, custom_key: nil,
+                     expires_at: nil, fresh: false, meta: nil)
```
Comment (ghost) [inline app/models/shortener/shortened_url.rb:29]: Instead of 'fresh', how about 'force' ?

Comment (mchaisse) [reply app/models/shortener/shortened_url.rb:29]: Since it is already part of the gem, we can open a new ticket to change the name.

**Comment by mchaisse** [comment]
@epmwgs PTAL

**Review by ghost — APPROVED** [review]

**Pull request merged**